lighthouse-core/gather/gather-runner.js

```diff
@@ -244,6 +244,9 @@ class GatherRunner {
       log.verbose('statusEnd', status);
     });
 
+    // Disable throttling so the afterPass analysis isn't throttled
+    pass = pass.then(_ => driver.setThrottling(options.flags, {useThrottling: false}));
```

Comment:
can you update the top outline to mention this as well? `endTrace (if requested) & endNetworkCollect & endThrottling` or something?